addons/zylann.editor_debugger/dock.tscn

```diff
@@ -1,41 +1,66 @@
-[gd_scene load_steps=2 format=3 uid="uid://d0sxhc6l66egc"]
+[gd_scene format=3 uid="uid://d0sxhc6l66egc"]
 
-[ext_resource type="Script" path="res://addons/zylann.editor_debugger/dock.gd" id="1"]
+[ext_resource type="Script" uid="uid://vpgnmtcfvsfr" path="res://addons/zylann.editor_debugger/dock.gd" id="1"]
 
-[node name="EditorDebugger" type="Control"]
+[sub_resource type="DPITexture" id="DPITexture_7dk6n"]
+_source = "<svg xmlns=\"http://www.w3.org/2000/svg\" width=\"16\" height=\"16\"><path fill=\"#e0e0e0\" d=\"M10.168 8.754a5 5 0 1 0-1.414 1.414l4.316 4.316 1.414-1.414zM6 3a3 3 0 0 1 0 6 3 3 0 0 1 0-6z\"/></svg>
+"
+saturation = 2.0
+color_map = {
+Color(1, 0.37254903, 0.37254903, 1): Color(1, 0.47, 0.42, 1),
+Color(0.37254903, 1, 0.5921569, 1): Color(0.45, 0.95, 0.5, 1),
+Color(1, 0.8666667, 0.39607844, 1): Color(0.83, 0.78, 0.62, 1)
+}
+
+[node name="EditorDebugger" type="Control" unique_id=1069202021]
 layout_mode = 3
 anchors_preset = 0
 offset_right = 238.0
 offset_bottom = 347.0
 script = ExtResource("1")
 
-[node name="PopupMenu" type="PopupMenu" parent="."]
+[node name="PopupMenu" type="PopupMenu" parent="." unique_id=1891947773]
```

Comment:
Looks like you saved the scene while its script was running. These items should not have to be in the scene file. Maybe `_ready` should have code that prevents logic from running if the scene is in the edited scene?